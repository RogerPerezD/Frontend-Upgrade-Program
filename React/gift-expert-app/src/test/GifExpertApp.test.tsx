import { shallow } from "enzyme";
import GifExpertApp from "../GifExpertApp";

describe('Test sobre GifExpertApp', () => {
    
    test('should show the component', () => {
        
        const wrapper = shallow( <GifExpertApp/>);

        expect( wrapper ).toMatchSnapshot();
    });
});