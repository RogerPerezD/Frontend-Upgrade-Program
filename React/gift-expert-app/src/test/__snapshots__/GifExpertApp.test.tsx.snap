// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test sobre GifExpertApp should show the component 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ul />
</Fragment>
`;
